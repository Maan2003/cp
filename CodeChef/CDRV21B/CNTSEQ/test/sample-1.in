8
4 6 8 2 3 3 2 2
1 2
2 3
3 4
3 5
5 7
2 6
1 8
10
3 1
3 2
3 3
3 4
3 5
2 1
2 2
2 3
4 1
4 3
