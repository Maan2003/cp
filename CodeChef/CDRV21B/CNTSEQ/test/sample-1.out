1
6
17
17
17
2
5
5
1
2
